test/integration/scan_test.go

```diff
@@ -1525,23 +1525,3 @@ func TestScanListWithFilters(t *testing.T) {
 	err, _ := executeCommand(t, args...)
 	assert.NilError(t, err, "")
 }
-
-func TestScanListWithBigLimitAndOtherFilters(t *testing.T) {
```

Comment:
why did you delete this test??

Reply:
By mistake. i revert the change in commit 44fba7a